mteb/tasks/Classification/eng/CanadaTaxCourtOutcomesLegalBenchClassification.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class CanadaTaxCourtOutcomesLegalBenchClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="CanadaTaxCourtOutcomesLegalBenchClassification",
+        description="Canada Tax Court Outcomes LegalBench Classification Dataset",
+        reference="https://huggingface.co/datasets/nguha/legalbench",
+        dataset={
+            "path": "nguha/legalbench",
+            "name": "canada_tax_court_outcomes",
+            "revision": "12ca3b695563788fead87a982ad1a068284413f4",
+        },
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["eng-Latn"],
+        main_score="accuracy",
+        date=("2023-08-23", "2023-08-23"),
+        form=None,
+        domains=["Legal"],
+        task_subtypes=None,
+        license="cc-by-nc-4.0",
+        socioeconomic_status="high",
+        annotations_creators="expert-annotated",
+        dialect=[],
```

Comment:
Seems like a dialect is needed here since it seems to be canadian

Reply:
You're right! Have added "en-CA" as the dialect for this dataset.